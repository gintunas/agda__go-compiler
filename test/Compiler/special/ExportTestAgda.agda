module ExportTestAgda where

open import Common.Prelude

itWorksText : String
itWorksText = "It works!"

{-# COMPILE GHC itWorksText as itWorksText #-}
