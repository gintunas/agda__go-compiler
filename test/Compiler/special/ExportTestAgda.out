EXECUTED_PROGRAM

ret > ExitSuccess
out > It works!
out >
